desktop/components/main_layout/templates/index.jade

```diff
@@ -1,6 +1,6 @@
 //- Override any locals with `append locals`
 block locals
-  - bodyClass = helpers && helpers.buildBodyClass ? helpers.buildBodyClass(sd, 'body-header-fixed') : '';
+  - bodyClass = helpers ? helpers.buildBodyClass(sd, 'body-header-fixed') : '';
```

Comment:
Feels weird to remove a safety check. But then again, `helpers` should only be undefined by accident, and I'm struggling to think of an easy pattern to have things like this fail a bit more gracefully.